### PR TITLE
Fixes in registration form validations and reg. list exports

### PR DESCRIPTION
**Pull request by nop33**

### CODE REVIEW

File: indico/MaKaC/PDFinterface/conference.py [hunk 1]
```diff
@@ -1630,6 +1631,7 @@ def getBody(self, story=None, indexedFlowable={}, level=1 ):
 
         for registration in self._regList:
             lp = []
+            lp.append(Paragraph(registration.friendly_id, text_format))
```
Comment (ThiefMaster) [inline indico/MaKaC/PDFinterface/conference.py]: It automatically converts the int to a string?

**Comment by ThiefMaster** [comment]
"book style" currently uses the `id` instead of the `friendly_id` of the registration

**Comment by ThiefMaster** [comment]
Multichoice is still broken. When modifying a registration the preselected values are not taken into account.

**Comment by nop33** [comment]
I fixed a bug that I found, and I cannot reproduce anymore your comment. Can you check if it's fine now?

**Pull request merged**